docs/table/tips_and_tricks.rst

```diff
@@ -0,0 +1,42 @@
+Tips and Tricks
+***************
+
+Using QTable to store user-defined units in a FITS file
+=======================================================
+
+A `~astropy.table.QTable` can be used to store data as :ref:`user-defined
+units <astropy:defining_units>` in a FITS file.
+
+.. EXAMPLE START: Combining Units and Quantities
+
+To output data to the current directory::
+
+  >>> import numpy as np
+  >>> from astropy import units as u
+  >>> from astropy.table import QTable
+
+  >>> lps = u.def_unit('Lps', u.L / u.s)
+  >>> lps_array = np.ones((5,5)) * lps
+  >>> qt = QTable()
+  >>> qt['speeds'] = lps_array 
+  >>> qt.write('Lps_output.fits',overwrite=True,format='fits')
+
+In order to parse and read the file contents, it is necessary to enable 
+the new defined unit as shown in :ref:`this example 
+<astropy:enabling_units>`::
+
+  >>> u.add_enabled_units(lps)
```

Comment:
There should first be an example of what happens if the table is read without enabling units. If anyone ever needs to look up this documentation then the presence of the exact error message might make it easier to find. Furthermore, if `astropy` is ever updated so that the explicit enabling of new units is no longer necessary then it would be good if that test fails so that updating the docs wouldn't be forgotten.